### PR TITLE
Added various ports

### PR DESCRIPTION
**Pull request by Dennisbonke**
This PR adds the following ports:

- `gettext`
- `findutils`
- `bzip2`

Furthermore, it adds a compatibility symlink from `gcc` to `cc`, so build systems that call `cc` can work.

### CODE REVIEW

File: bootstrap.yml [hunk 1]
```diff
@@ -2673,6 +2674,59 @@ packages:
...
+      git: 'https://git.savannah.gnu.org/git/gettext.git'
+      tag: 'v0.20.2'
+      tools_required:
+        - host-autoconf-v2.69
+        - host-automake-v1.15
+        - host-libtool
+        - host-pkg-config
+      regenerate:
+        - args: 'sed -i s/"SUBDIRS = doc intl intl-java intl-csharp gnulib-lib \$(SUBDIR_libasprintf)\ src po man m4 tests"/"SUBDIRS = intl intl-java intl-csharp gnulib-lib \$(SUBDIR_libasprintf)\ src po m4 tests"/ @THIS_SOURCE_DIR@/gettext-runtime/Makefile.am'
+        - args: 'sed -i s/"SUBDIRS = intl gnulib-lib libgrep src libgettextpo po its projects styles emacs misc man m4 tests system-tests gnulib-tests examples doc"/"SUBDIRS = intl gnulib-lib libgrep src libgettextpo po its projects styles emacs misc m4 tests system-tests gnulib-tests examples"/ @THIS_SOURCE_DIR@/gettext-tools/Makefile.am'
```
Comment (avdgrinten) [inline bootstrap.yml:2690]: Maybe add a comment on what this does.

File: bootstrap.yml [hunk 2]
```diff
@@ -2768,6 +2822,59 @@ packages:
...
+    source:
+      subdir: ports
+      url: 'https://www.sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz'
+      format: 'tar.gz'
+      extract_path: 'bzip2-1.0.8'
+    tools_required:
+      - system-gcc
+    configure:
+      - args: ['cp', '-r', '@THIS_SOURCE_DIR@/.', '@THIS_BUILD_DIR@']
+      - args: 'sed -i s/"all: libbz2.a bzip2 bzip2recover test"/"all: libbz2.a bzip2 bzip2recover"/ @THIS_BUILD_DIR@/Makefile'
```
Comment (avdgrinten) [inline bootstrap.yml:2871]: Dito.

File: bootstrap.yml [hunk 3]
```diff
@@ -2768,6 +2822,59 @@ packages:
...
+    build:
+      - args: ['make', 'CC=x86_64-managarm-gcc', 'CFLAGS=-fPIC', '-f', 'Makefile-libbz2_so']
+      - args: ['make', 'clean']
+      - args: ['make', 'CC=x86_64-managarm-gcc', 'CFLAGS=-fPIC', '-j@PARALLELISM@']
```
Comment (avdgrinten) [inline bootstrap.yml:2875]: Why do we run `make` -> `make clean` -> `make` again?

File: scripts/prepare-sysroot [hunk 4]
```diff
@@ -10,6 +10,9 @@ mkdir -p system-root/root
 mkdir -p system-root/usr
 mkdir -p system-root/usr/share
 mkdir -p system-root/usr/share/X11
+mkdir -p system-root/var
+mkdir -p system-root/var/lib
+mkdir -p system-root/var/lib/locate
```
Comment (avdgrinten) [inline scripts/prepare-sysroot:15]: This should be added to the `locate` package.

File: bootstrap.yml (hunk 3 above)
Comment (Dennisbonke) [reply bootstrap.yml:2900]: we run make on the makefile called Makefile-libbz2_so, so everything builds against a shared libbz2, this is in line with the LFS instructions found [here](http://www.linuxfromscratch.org/lfs/view/systemd/chapter06/bzip2.html)

**Pull request merged**